step/src/main/webapp/chat.jsp

```diff
@@ -167,25 +177,8 @@ limitations under the License.
                 }
             }
 
-            function setImage(image) {
-                const imageContainer = document.getElementById('profile-pic');
-                userIcon = document.createElement('IMG');
-                
-                if (image === "") {
-                userIcon.setAttribute('src', "avatar.png");
-                } else {
-                userIcon.setAttribute('src', "/serve?key=" + image);
-                }
-
-                userIcon.setAttribute('alt', "Profile picture.");
-                imageContainer.appendChild(userIcon);
-            }
-
-            async function grabBlobURL(){
-                const blobURL = await fetch("/image-upload").then((response) => {return response.text();});
-                const myForm = document.getElementById("image-form");
-                myForm.action = blobURL;
-            }
+            document.cookie = 'same-site-cookie=foo; SameSite=None; Secure';
+            document.cookie = 'cross-site-cookie=bar; SameSite=None; Secure';
```

Comment:
Is this needed? Foo and bar usually aren't names for permanent things

Reply:
Also, what are we using cookies for?